src/groups/group.jl

```diff
@@ -957,25 +976,25 @@ struct MultiplicationOperation <: AbstractGroupOperation end
 
 const MultiplicationGroup = AbstractGroupManifold{𝔽,MultiplicationOperation} where {𝔽}
 
-*(e::Identity{G}) where {G<:MultiplicationGroup} = e
-*(::Identity{G}, p) where {G<:MultiplicationGroup} = p
-*(p, ::Identity{G}) where {G<:MultiplicationGroup} = p
-*(e::E, ::E) where {G<:MultiplicationGroup,E<:Identity{G}} = e
-*(::Identity{<:MultiplicationGroup}, e::Identity{<:AdditionGroup}) = e
+Base.:*(e::Identity{G}) where {G<:MultiplicationGroup} = e
```

Comment:
I wasn't aware of the `:` notation – and: We now prepend the package for every foreign function definition that we extend? Because then we might miss `Base.exp` and `Base.log` (mainly when defining more docstrings, since we implement `exp!` and `log!`)?

Reply:
I don't follow. `exp!` and `log!` are unique to our package. We don't need to use `Base.exp` and `Base.log` when defining docstrings, as that doesn't extend the function. And if we accidentally implement an `exp` (without the prepend), the test suite raises an error asking us if we meant to import `Base.exp`. Is there another scenario where this could break things?

Reply:
Ah, I wasn't aware, that this does not hold for docstrings.